src/ui/impl/webview/webview.cpp

```diff
@@ -206,6 +206,11 @@ namespace Soundux::Objects
                         tray->getEntries().at(1)->setText(translations.hide);
                     }
                 }));
+                if (hideOnStartup)
```

Comment:
This check breaks translations?
setText with the translation should always be called